style50/__init__.py

```diff
@@ -1,3 +1,20 @@
+from pkg_resources import get_distribution, DistributionNotFound
+import os
+
+# https://stackoverflow.com/questions/17583443/what-is-the-correct-way-to-share-package-version-with-setup-py-and-the-package
+try:
+    _dist = get_distribution("style50")
+    # Normalize path for cross-OS compatibility
+    _dist_loc = os.path.normcase(_dist.location)
+    _here = os.path.normcase(__file__)
+    if not _here.startswith(os.path.join(_dist_loc, "style50")):
+        # this version is not installed, but another version is.
```

Comment:
Sorry about that, done! (although making that commit automatically dismissed your review)